### ISSUE TITLE
Ignore files in the folder

### ISSUE THREAD

**Issue by pyronicide** [vapor-ware/ksync#127]
A lot of the time, you don't want to sync all the files in a folder (just some of them, ex. `.git`). Make it possible to ignore in the specification.

**Comment by pyronicide** [comment]
https://docs.syncthing.net/users/ignoring.html

**Comment by pyronicide** [comment]
@timfallmk what do you think the best way to integrate this in is?

- Add it to `create`
- Make a `.ksync-ignore` file

**Comment by timfallmk** [comment]
I would go with both actually. Similar structure to `.gitignore` syntax (including with `!`)

**Comment by snickell** [comment]
It'd be great if I could feed existing .ignore files in, e.g. for code-editing cases I suspect the combination of .dockerignore and .gitignore is perfect as a starting point.

**Comment by timfallmk** [comment]
@snickell you _should_™ be able to use standard `.ignore` syntax files labeled as `.stignore` (from the above syncthing link) and that should get you 90% of the way. I don't really remember but I suspect that's why this wasn't further worked on.

**Comment by alok87** [comment]
Can i work on this? ✋  @grampelberg

**Comment by grampelberg** [comment]
@alok87 yeah! That'd be awesome. It should be some automation around writing `.stignore` files.

**Comment by timfallmk** [comment]
@alok87 Happy to have it!

**Comment by alok87** [comment]
adding `.stignore` file makes syncthing process running locally to ignore the patterns mentioned in it. What is that is required in this issue. Please suggest.

**Comment by igoooor** [comment]
Do we have an ETA for this feature? or anyone found an alternative in the meantime?

**Comment by grampelberg** [comment]
@igoooor did you try `.stignore` yet?

**Comment by igoooor** [comment]
@grampelberg yes but I don't manage to make it work.
Let's take this example:
```
ksync create -p my-pod $(pwd)/docroot /home/www-data/docroot
```

I want to ignore `docroot/vendor` folder
Should I create a `.stignore` in my current directory `$(pwd)` with content `docroot/vendor` 
or in the directory I'm syncing `$(pwd)/docroot` and the content `vendor`
I tried both and the `docroot/vendor` is always being synced

**Comment by igoooor** [comment]
so, it seems to work, only if the `.stignore` is inside the directory I'm syncing, so not in the current directory. and the file `.stignore` has to be present in the container.
Is there anyway to make it work with `.stignore` inside of the current directory instead?

**Comment by grampelberg** [comment]
I suspect this is because `.stignore` is evaluated on the side that the syncing occurs - eg. it needs to be in the container to prevent `vendor` inside the container from syncing back. Have you taken a peek at the syncthing docs? They might have a way to address this.
